README.md

```diff
@@ -1,2 +1,8 @@
 # flux-uploader
 Flux SDK app that can upload to data keys and visualize geometry.
+
+To run locally:
+1. From command line in the flux-uploader folder, run npm install express
```

Comment:
You can just say 'npm install', not 'npm install express', express is already specified as a dependency in package.json.